contrib/babelfishpg_tsql/src/pltsql_coerce.c

```diff
@@ -1403,6 +1438,7 @@ resolve_numeric_typmod_from_exp(Plan *plan, Node *expr, bool *found)
 
 				if (var->vartypmod == -1)
 				{
+					int32 		fixlen_default_typmod;
```

Comment:
Referencing from previous PRs as well, please declare variables in the scope where we need it